network/network-tutorial-overlay.md

```diff
@@ -401,52 +399,47 @@ The example uses Linux hosts, but the same commands work on Windows.
     uw9etrdymism        test-net            overlay             swarm
     ```
 
-5.  Remember that you created `alpine1` from `host1` and `alpine2` from `host2`.
-    Now, attach to `alpine2` from `host1`:
+7.  On `host1`, ping `alpine2` within the `/bin/sh` session of `alpine1`:
 
     ```bash
-    (host1) $ docker container attach alpine2
+    / # ping -c 2 alpine2
+    PING alpine2 (10.0.0.5): 56 data bytes
+    64 bytes from 10.0.0.5: seq=0 ttl=64 time=0.729 ms
+    64 bytes from 10.0.0.5: seq=1 ttl=64 time=0.601 ms
 
-    #
+    --- alpine2 ping statistics ---
+    2 packets transmitted, 2 packets received, 0% packet loss
+    round-trip min/avg/max = 0.601/0.665/0.729 ms
     ```
 
-    Automatic service discovery worked between two containers across the overlay
-    network!
+    The two containers can communicate using the overlay network connecting
+    `host1` and `host2`.
 
-    Within the attached session, try pinging `alpine1` from `alpine2`:
+8.  On `host1`, close the `alpine1` session (which stops the container):
 
     ```bash
-    # ping -c 2 alpine1
-
-    PING alpine1 (10.0.0.2): 56 data bytes
-    64 bytes from 10.0.0.2: seq=0 ttl=64 time=0.523 ms
-    64 bytes from 10.0.0.2: seq=1 ttl=64 time=0.547 ms
-
-    --- alpine1 ping statistics ---
-    2 packets transmitted, 2 packets received, 0% packet loss
-    round-trip min/avg/max = 0.523/0.535/0.547 ms
+    / # exit
     ```
 
-    This proves that the two containers can communicate with each other using
-    the overlay network which is connecting `host1` and `host2`.
+9.  Clean up your containers and networks:
 
-    Detach from `alpine2` using the `CTRL` + `P` `CTRL` + `Q` sequence.
+    Because the Docker daemons operate independently and these are standalone
+    containers, you must run the commands on the individual hosts.
 
-6.  Stop the containers and remove `test-net` from each host. Because the Docker
-    daemons are operating independently and these are standalone containers, you
-    need to run the commands on the individual hosts.
+    a.  On `host1`, remove `alpine1` and `test-net`:
 
-    ```bash
-    (host1) $ docker container stop alpine1
-            $ docker container rm alpine1
-            $ docker network rm test-net
-    ```
+        ```bash
+        (host1) $ docker container rm alpine1
+                $ docker network rm test-net
+        ```
 
-    ```bash
-    (host2) $ docker container stop alpine2
-            $ docker container rm alpine2
-            $ docker network rm test-net
-    ```
+    b.  On `host2`, stop and remove `alpine2` and  remove `test-net`:
+
+        ```bash
+        (host2) $ docker container stop alpine2
+                $ docker container rm alpine2
+                $ docker network rm test-net
+        ```
 
 ## Communicate between a container and a swarm service
```

Comment:
Since you're working on this, you might also want to address this issue #6407

Reply:
If you deploy `alpine1` as detached, then you have to attach it again to run `ping`.  So I'm not going to detach it. 

With `alpine2`, I was not pinging back so there was no need to attach (or in this case not detach). I was demonstrating a more deliberate use of the flags ... which is also good to know.

Reply:
Ps. I was going to address that one in a separate PR ... we'll see.